enroll/service.go

```diff
@@ -98,3 +103,58 @@ func (svc service) Enroll() (Profile, error) {
 
 	return *profile, nil
 }
+
+type service struct {
+	Url           string
+	SCEPUrl       string
+	SCEPChallenge string
+	SCEPSubject   [][][]string
+	Topic         string // APNS Topic for MDM notifications
+}
+
+func (svc service) Enroll() (Profile, error) {
```

Comment:
I'm confused. a bit earlier you added a `Enroll` method with a `ctx` argument. This one is different?